src/main/java/frc/robot/commands/drive/DriveCommandBase.java

```diff
@@ -39,43 +42,43 @@ public void execute() {
     swerveDrive.addPoseEstimatorSwerveMeasurement();
     vision.setHeadingInfo(
         swerveDrive.getPose().getRotation().getDegrees(), swerveDrive.getGyroRate());
-    calculatePoseFromLimelight(VisionConstants.SHOOTER_LIMELIGHT_NUMBER);
-    calculatePoseFromLimelight(VisionConstants.FRONT_LEFT_LIMELIGHT_NUMBER);
-    calculatePoseFromLimelight(VisionConstants.FRONT_RIGHT_LIMELIGHT_NUMBER);
+    calculatePoseFromLimelight(Limelight.SHOOTER);
```

Comment:
what is Limelight.SHOOTER? Is Limelight the constants class now?

Reply:
its an enum within visionconstants

Reply:
nice, this is better

Reply:
Bro is not cooking with these comments